python/semantic_kernel/core_skills/conversation_summary_skill.py

```diff
@@ -44,15 +45,20 @@ def __init__(self, kernel: Kernel):
         input_description="A long conversation transcript.",
     )
     async def summarize_conversation_async(
-        self, input: str, context: SKContext
-    ) -> SKContext:
+        self, input: str, context: "SKContext"
+    ) -> "SKContext":
         """
         Given a long conversation transcript, summarize the conversation.
 
         :param input: A long conversation transcript.
         :param context: The SKContext for function execution.
         :return: SKContext with the summarized conversation result.
         """
+        from semantic_kernel.text import text_chunker
```

Comment:
Why are you putting these imports at the function level instead of module level?

Reply:
We have a number of circular imports in the code that are bad practice and that Pydantic will not accommodate. This is the least invasive method of addressing the problem.